### DISCUSSION TITLE
Dark theme design customization is not available.

### DISCUSSION THREAD

**Discussion by ThilinaV98** [mynaparrot/plugNmeet-server#342] [Q&A] [answered]
We can customized the colors of the light theme using config.js or Global Changes using CSS. but there is no option to customized the colors of the dark theme. 
Example - if i use 
window.DESIGN_CUSTOMIZATION = `{
  "header_bg_color": "#FFFFFF",
}`;
in light mode there is no problem h2.header-title is visible because in light theme h2.header-title color is black but when change it to dark mode it automatically changes the  h2.header-title color to white. In this case header_bg_color is white and h2.header-title color also white then h2.header-title is not visible. 

How to customized the dark theme colors?

**Comment by jibon57** [comment, marked as answer]
Yes, at present not possible & we don't have plan to support it but contribution is welcome.

**Reply by jibon57** [reply]
But you can use custom css file for this purpose

**Reply by ThilinaV98** [reply]
How can I use custom css file for it ?

**Reply by jibon57** [reply]
https://github.com/mynaparrot/plugNmeet-client/blob/0e51e32d4a5584886b44cae721117ec15ad7826d/src/assets/config_sample.js#L46

https://www.plugnmeet.org/docs/developer-guide/design-customisation
